tests/how_to_test_apps.rs

```diff
@@ -0,0 +1,133 @@
+#![allow(missing_docs)]
+/// Demonstrates simple integration testing of Bevy applications.
```

Comment:
These should be `//!` comments, since they're at the start of the file.

Reply:
Argh, I had it that way and changed it. Have now read the rustdoc book in penance :laughing: